web/cobrands/northamptonshire/assets.js

```diff
@@ -618,4 +619,33 @@ function check_emergency() {
 }
 $(fixmystreet).on('report_new:category_change', check_emergency);
 
+function ncc_check_streetlights() {
+    var emergency_state = ncc_is_emergency_category();
+
+    if (emergency_state.relevant || fixmystreet.body_overrides.get_only_send() == 'Highways England') {
+        $('#ncc_streetlights').remove();
+        return;
+    }
+
+    var relevant_body = OpenLayers.Util.indexOf(fixmystreet.bodies, northants_defaults.body) > -1;
+    var relevant_cat = $('#form_category').val() == 'Street lighting';
+    var relevant = relevant_body && relevant_cat;
+    var currently_shown = !!$('#ncc_streetlights').length;
+
+  console.log(relevant, currently_shown);
+    if (!relevant) {
+        $('#ncc_streetlights').remove();
+        enable_report_form();
+        return;
+    }
+
+    if (!currently_shown) {
```

Comment:
Might be nicer (slash easier for me to include in refactor :) ) if it was consistent with the other calls which return first if (relevant === currently_shown). Then this line wouldn't be needed, I think? Unless there was a reason for not doing that.